tests/network-ovn

```diff
@@ -1,25 +1,31 @@
 #!/bin/bash
 set -eux
 
-# Install dependencies
-install_deps ovn-host ovn-central bind9-dnsutils jq
-
 # Install LXD
 install_lxd
 
-# Configure OVN.
-ovs-vsctl set open_vswitch . \
-  external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
-  external_ids:ovn-encap-type=geneve \
-  external_ids:ovn-encap-ip=127.0.0.1
-
-# Empty controller log so ACL log checks are consistent over repeat runs.
-echo "" > /var/log/ovn/ovn-controller.log
-
 # Configure LXD.
 lxc project switch default
 lxc storage create default zfs
 
+if [ $# -eq 0 ]; then
+    # Install dependencies
+    install_deps ovn-host ovn-central bind9-dnsutils jq
+
+    # Configure OVN.
+    ovs-vsctl set open_vswitch . \
+        external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
+        external_ids:ovn-encap-type=geneve \
+        external_ids:ovn-encap-ip=127.0.0.1
+
+    # Empty controller log so ACL log checks are consistent over repeat runs.
+    echo "" > /var/log/ovn/ovn-controller.log
+elif [ "$1" == "microovn" ]; then
```

Comment:
`==` inside simple `[]` is non-standard AFAIK, `[ "${1}" = "microovn" ]` is what we use elsewhere. https://www.gnu.org/software/bash/manual/html_node/Conditional-Constructs.html

Reply:
fixed